packages/amplication-client/src/Layout/InnerTabLink.scss

```diff
@@ -3,6 +3,7 @@
 @mixin active-link {
   background-color: var(--black10);
   color: var(--black100);
+  padding: var(--default-spacing-small);
```

Comment:
`InnerTabLink` is a component that is used in a lot of places. Are you sure that this change doesn't affect anywhere else?

Reply:
Kinda sure here is screenshot with `commitListItem` and another place that use `InnerTabLink`

<img width="302" alt="Screen Shot 2565-10-28 at 00 20 33" src="https://user-images.githubusercontent.com/30198386/198357575-87f3eb80-fe21-47e3-9f6a-1e0fdae654ce.png">
<img width="1286" alt="Screen Shot 2565-10-28 at 00 22 22" src="https://user-images.githubusercontent.com/30198386/198357629-b9638cde-9f32-4b3a-9ab5-07fa26a55f7e.png">

Reply:
or we should add width: 100% to `.inner-tab-link` instead?